tests/integration/values.rs

```diff
@@ -0,0 +1,155 @@
+use predicates::prelude::*;
+
+use crate::helpers::project::binary;
+use crate::helpers::project_builder::tmp_dir;
+
+use assert_cmd::prelude::*;
+use indoc::indoc;
+
+#[test]
+fn it_accepts_template_values_file() {
+    let template = tmp_dir()
+        .file(
+            "my-values.toml",
+            indoc! {r#"
+                [values]
+                my_value = "content of my-value"
+            "#},
```

Comment:
I think this indentation level here is not correct. It should be on the same as in the lines above right?

Reply:
I guess thats a matter of taste - personally I like it this way, as it's all the same parameter for the `indoc!` macro, separate from the filename parameter preceeding it. The `indoc!` macro is nice because it removes all the extra fluff spacing thats there just to make it easier to scan through the code.

I'd prefer to keep it this way if possible.

Reply:
sure, fine with me